godtools/Views/TractElements/TractImage.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ImageContent.swift
+//  godtools
+//
+//  Created by Devserker on 5/11/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+class TractImage: BaseTractElement {
+    
+    static let xMarginConstant: CGFloat = 8.0
+    static let yMarginConstant: CGFloat = 16.0
+    
+    var align = "center"
+    
+    var xPosition: CGFloat {
+        return CGFloat(0.0)
+    }
+    var yPosition: CGFloat {
+        return self.yStartPosition + TractImage.yMarginConstant
+    }
+    override var width: CGFloat {
+        return (self.parent?.width)! - (self.xPosition * CGFloat(2))
+    }
+    
+    var imageView = UIImageView()
+    
+    override func setupView(properties: [String: Any]) {
+        self.imageView = UIImageView(image: loadImage(properties: properties))
+        var width = self.imageView.frame.size.width
+        var height = self.imageView.frame.size.height
+        var xPosition = CGFloat(0.0)
+        let yPosition = CGFloat(0.0)
+        
+        if width > self.width {
```

Comment:
If the image is wider than the view, I agree we should down scale it like you have here. Do we still want to make sure it has a margin on the sides too?